data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1680 @@
+			branch resistance
+				has "Resistance: Questions: done"
+			`	Freya is atop a ladder near the transmitter when she sees you, coming down and running to the entrance. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she whistles to the teams, dismissing them for lunch. When you're alone, you give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, and about who House Bebliss is, handing her the suitcase when you're done.`
+				goto suitcase
+			label resistance
+			`	Freya is atop a ladder near the transmitter when she sees you, coming down and running to the entrance. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she whistles to the teams, dismissing them for lunch. When you're alone, you give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, the Lunarium Redoubt and their intentions to overthrow the Heliarchy, and the part House Bebliss plays in that, handing her the suitcase when you're done.`
+			label suitcase
+			`	"Well, at least these ones are asking if they can come invade us," she jokes as she lays the suitcase on a table, opening it. "Three alien species, right on the Free Worlds' backdoor huh?" She begins to read the message House Bebliss prepared for her, frowning right at the start. "You didn't write this, did you?" You shake your head. "Thought so. The words are... jumbled?"`
+			choice
+				`	"Oh, that. You get used to it."`
+				`	"It's not so bad when you're talking with them in person."`
+			`	She gives the message an odd look, lets out an "If you say so," and continues to read it. "If this 'House Bebliss' is as advanced as they claim in this, I don't see how I could help them with anything, but they're offering to share their knowledge with me in return, so I won't complain. If nothing else it'll be an experience, working with giant spiders."`
+			choice
+				`	"So you're fine with letting them study the Pug artifacts with you?"`
+				`	"I can go back and tell them you've agreed then?"`
+			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet>, you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachis and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`
```

Comment:
Is the use of 'Arachis' intentional here? I remember the plural of Arachi just being Arachi.
```suggestion
			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet> - you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachis and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`
```

Reply:
Wasn't intentional no.